githubnotes.txt

```diff
@@ -0,0 +1,199 @@
+What is Github
```

Comment:
Do some markup formatting. 
https://help.github.com/articles/basic-writing-and-formatting-syntax/